### DISCUSSION TITLE
请问我这个代码哪里错了？为什么运行结果什么都没有，也不报错，直接跳转到了新的一行

### DISCUSSION THREAD

**Discussion by bunny-sadako** [datawhalechina/learn-python-the-smart-way-v2#21] [提问答疑] [answered]
W11 python3.9
<img width="1280" alt="image" src="https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/128591902/745362ea-4b4c-48a9-88c8-1c09e1dcbc97">

**Comment by anine09** [comment, marked as answer]
作业请在[作业系统](https://hydro.ac/d/datawhale_p2s/)中提交，你遇到的问题在[课程安排](https://www.bilibili.com/video/BV1x44y1w76d)中有答案。

**鉴于你没有按照[提问规则](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Question/question/)提问，且不是初次发生，我们将锁定这个问题。**
